guides/source/configuring.md

```diff
@@ -217,6 +217,22 @@ end
 Controls whether requests should be handled concurrently. This should only
 be set to `false` if application code is not thread safe. Defaults to `true`.
 
+#### `config.app_version`
+
+Configures application version management. This allows Rails to track and expose version and environment information through various endpoints. Accepts an `ActiveSupport::OrderedOptions` object with the following settings:
+
+* `enabled`: Enable or disable version management (default: `true`)
+* `add_headers`: Add version and environment headers to HTTP responses (default: `true`)
```

Comment:
This is maybe a old opinion but exposing version information to the public is probably not preferrable in some cases to make it the default